### PR TITLE
feat(metrics): added basic auth prometheus config

### PR DESCRIPTION
**Pull request by serge1peshcoff**

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -95,3 +95,7 @@ PATH_OMS_ELASTIC=oms-elastic/docker/
...
+
+# You can generate this with `htpasswd -nbB admin "<password>"`
+METRICS_CREDENTIALS=$2y$05$TCEmoztTS37.euXQAXzC/.yQ6D/LtL4oxNMN.bgPtxTRhxF.Trv6a
```
Comment (linuxbandit) [inline .env.example:100]: this doesn't look like `5ecr3t` (as all passwords above)

Comment (serge1peshcoff) [reply .env.example:100]: yeah, this is probably apparently `htpasswd -nbB admin "<password>"` yields different results each time it's executed, as well as bcrypt does for example

Comment (linuxbandit) [reply .env.example:100]: how is it possible? we didn't specify any salt or anything, how can a hash be different

Comment (serge1peshcoff) [reply .env.example:100]:
```
sh-3.2$ htpasswd -nbB admin "5ecr3t"
admin:$2y$05$rWZihh0QG4SPCzO.X0W2Ku7.pywS5bM5XxLDVL/cdH4qc2YuMmCbK

sh-3.2$ htpasswd -nbB admin "5ecr3t"
admin:$2y$05$0mmCbqJFI2XTyQ2QYwXQVO1l5RSzJ5jnKJsHR10loCKXSMthPl8Q.

sh-3.2$ htpasswd -nbB admin "5ecr3t"
admin:$2y$05$KicylyX90m48KBQS1YfGC.k0VTaLo4wcJDXnlfHR0qtVvlgQUouQ.

sh-3.2$ htpasswd -nbB admin "5ecr3t"
admin:$2y$05$wHi3QbUg5LDm6AAjPHHl8e8U45XpoqfXiqf3ouYhVvB1rdgkPVX7C

sh-3.2$ htpasswd -nbB admin "5ecr3t"
admin:$2y$05$RSamQzeoXaK72VFMa8AZOOthuKTr2Ss37mpJ.NFBk6QxmKhM0KYqu

sh-3.2$ htpasswd -nbB admin "5ecr3t"
admin:$2y$05$.evCohCYAAc0OBIWA9pi5uM.rWw584y9dyflEvnTg2BsXMrIoKlVK

sh-3.2$ htpasswd -nbB admin "5ecr3t"
admin:$2y$05$MA0.QNK1DpTLcIW0ASeF4uKxTD0CLvAylp7SfZK0LO7X08EyKT67y

sh-3.2$ htpasswd -nbB admin "5ecr3t"
admin:$2y$05$5pUsBw/blABGlH5RGi9pYekRpHnvz1a.MCn6dJF9u1LSCI.NJU7XK

sh-3.2$ htpasswd -nbB admin "5ecr3t"
admin:$2y$05$gW7PsdwA7MCISCkGxbDu8.zmgXCfp9/Kd2ybN/89w7DAdifHyk8LO

sh-3.2$ htpasswd -nbB admin "5ecr3t"
admin:$2y$05$QKtiNr7o87dLjOeqJiZznuyMa82/9.g7skcrottWGGai0lvRbFEUW
```

no idea, but it is different lol

Comment (WikiRik) [reply .env.example:100]: It uses the crypt_blowfish algorithm according to; https://httpd.apache.org/docs/2.4/misc/password_encryptions.html
So I guess that's the reason why the hashes are different

**Comment by serge1peshcoff** [comment]
@linuxbandit ping?

**Review by linuxbandit — APPROVED** [review]

**Comment by linuxbandit** [comment]
sorry, done

**Pull request merged**